src/braket/device_schema/quera/quera_ahs_paradigm_properties_v1.py

```diff
@@ -124,21 +124,180 @@ class PerformanceLattice(BaseModel):
+            coherent evolution as a result of thermal motion. (measured in meters)
+        fillingErrorTypical (Annotated[Decimal, Field(ge=0, le=1)]): Typical probability of failing
+            to occupy a site specified by user as 'filled'. These probabilities are dependent on
+            the pattern and site position within the pattern. Normalized to 1.
+        fillingErrorWorst (Annotated[Decimal, Field(ge=0, le=1)]): Worst-case probability of
+            failing to occupy a site specified by user as 'filled'. Upper bound that includes the
+            pattern-dependence and site position dependence. Normalized to 1.
+        vacancyErrorTypical (Annotated[Decimal, Field(ge=0, le=1)]): Typical probability of
+            erroneously filling a site specified by user as 'unfilled'. These probabilities can be
+            dependent on the pattern and site position within the pattern, and can change slightly
+            with time. Normalized to 1.
+        vacancyErrorWorst (Annotated[Decimal, Field(ge=0, le=1)]): Worst-case probability of
+            erroneously filling a site specified by user as 'unfilled'. Upper bound that includes
+            the pattern-dependence, site position dependence and time-variation of this
+            probability. Normalized to 1.
+        atomLossProbabilityTypical (Annotated[Decimal, Field(ge=0, le=1)]): Typical probability of
+            atom loss from a filled site between the first and second image. These probabilities
+            can be dependent on the pattern and site position within the pattern, and can change
+            slightly with time. Normalized to 1.
+        atomLossProbabilityWorst (Annotated[Decimal, Field(ge=0, le=1)]): Worst-case probability of
+            atom loss from a filled site between the first and second image. Upper bound that
+            includes the pattern-dependence, site position dependence and time-variation of this
+            probability. Normalized to 1.
+        atomCaptureProbabilityTypical (Annotated[Decimal, Field(ge=0, le=1)]): Typical probability
+            of atom capture into an empty site between the first and second image. These
+            probabilities can be dependent on the pattern and site position within the pattern, and
+            can change slightly with time. Normalized to 1.
+        atomCaptureProbabilityWorst (Annotated[Decimal, Field(ge=0, le=1)]): Worst-case probability
+            of atom capture from a filled site between the first and second image. Upper bound that
+            includes the pattern-dependence, site position dependence and time-variation of this
+            probability. Normalized to 1.
+        atomDetectionErrorFalsePositiveTypical (Annotated[Decimal, Field(ge=0, le=1)]): Typical
+            probability of a false-positive atom detection error. These probabilities can be
+            dependent on the pattern and site position within the pattern, and can change slightly
+    with time. Normalized to 1.
```

Comment:
formatting issue here

Reply:
Do you know why the linters are not catching this? The check code format passes here.

Reply:
We do not verify the RST format with the current docstring linters